materials/03-remote-hosting.qmd

```diff
@@ -2,4 +2,203 @@
 title: "3. Remote Code Hosting and GitHub"
 ---
 
-## 
+```{r include=FALSE}
+knitr::opts_chunk$set(eval = FALSE)
+```
+
+Version control really comes into its own when we begin to collaborate with other people. We already have most of the machinery we need to do this ; the only thing missing is to copy changes from one repository to another.
+
+Systems like Git allow us to move work between two repositories. The repositories can live on separate computers or network locations. In practice, we often use one copy as a central hub, and keep it on the web rather than on someone’s laptop. Most programmers use a hosting service like GitHub to hold the central repositories; and this is true for many BC Government employees.
+
+If you recall, in the pre-workshop, you created a GitHub account and associated it with your GitHub handle (or ID). You also configured your computer with a token (PAT) in a way that should seamlessly allow us to to create a remote repository on GitHub that will be linked to our local repository. Then we will share the changes we’ve made to our current project with the world.
+
+## 3.1 **Creating a Remote Repository**
+
+1.  Log in to GitHub. Your personal dashboard is the first page you'll see when you sign in on GitHub.
+
+    Tip: To access your personal dashboard once you're signed in, click the invertocat ![](images/github-mark.png){width="13"} logo in the upper-left corner of any page on GitHub.
+
+2.  Click on the `+` icon in the top right corner of your personal dashboard, then `New Repository`
+
+3.  Name your repository “planets”. Further down the page are other options available to us that can be toggled to suit the repository's purpose. For this workshop it's fine to create a "Public" repository however since this repository will be connected to a local repository, it needs to be empty. Therefore, leave “Add a README file” unchecked, and select “None” as options for both “Add .gitignore” and “Add a license.”
```

Comment:
```suggestion
3.  Name your repository “planets”. Further down the page are other options available to us that can be toggled to suit the repository's purpose. For this workshop it's fine to create a "Public" repository. However, as this repository will be connected to a local repository, it needs to be empty. Therefore, leave “Add a README file” unchecked, and select “None” as options for both “Add .gitignore” and “Add a license.”
```

Reply:
Does it make more sense to teach how to start from a remote repo (because having the README.md and license files are usually wanted/needed for bcgov repos anyways), and clone that, instead of trying to match it up with a local one that already exists?